qiskit/transpiler/passes/calibration/rzx_builder.py

```diff
@@ -90,6 +95,8 @@ def __init__(
 
         self._inst_map = instruction_schedule_map
         self._verbose = verbose
+        if target:
+            self._inst_map = target.instruction_schedule_map()
```

Comment:
Alternatively you can store target instead of the legacy instmap if available, or I can do it in follow up.

Reply:
Yeah, I was thinking we could optimize some of these passes to work natively with a target in the future. For this PR just having the target option is sufficient I think because my goal for 0.24.0 is to update all the preset pass managers to work solely with a target so we can start to unify the transpiler passes to only need a target in the future.